### DISCUSSION TITLE
undefined is not an object (evaluating '_$$_REQUIRE(_dependencyMap[10], "./core").isConfigured')

### DISCUSSION THREAD

**Discussion by ReshmaOza** [software-mansion/react-native-reanimated#5763] [Q&A] [answered]
I am facing this issue undefined is not an object (evaluating '_$$_REQUIRE(_dependencyMap[10], "./core").isConfigured')
![Screenshot_1709726383](https://github.com/software-mansion/react-native-reanimated/assets/86316018/0ec4ec04-3661-4d24-aa3e-df7e050525fe)

**Comment by MatiPl01** [comment]
Hi. Could you share your code? It's hard to tell what is causing the issue without seeing code that is executed.

**Comment by ReshmaOza** [comment, marked as answer]
Its solved now , its reanimation plugin issue , i downgrade version and remove ^ its starts working
